tests/integration/users.test.js

```diff
@@ -87,10 +88,10 @@ describe('POST /users', () => {
     });
```

Comment:
Ligne 57 qu penses-tu de renommer le it: `it should create a user`

Reply:
done